loadgen/manifest.yaml

```diff
@@ -0,0 +1,22 @@
+dist:
```

Comment:
Do we want to keep this distribution manifest? Should we add the `loadgenreceiver` into the elastic-components distribution instead https://github.com/elastic/opentelemetry-collector-components/blob/main/distributions/elastic-components/manifest.yaml?

Reply:
It is used to generate a purpose-built loadgencol binary under _loadgenbuild. Although in the near future I'm thinking about not using ocb and just wrap and distribute it with out own main function, so it may irrelevant soon.

Are you proposing to just add it to collector components, and to run loadgen it will just be using a specific config with this distro?

Reply:
> Are you proposing to just add it to collector components, and to run loadgen it will just be using a specific config with this distro?

Exactly, I would reuse the already available distribution. That distribution is used in the CI to ensure a binary can be built from the linked components. And extend or add a new config file with the loadgen receiver https://github.com/elastic/opentelemetry-collector-components/blob/main/distributions/elastic-components/config.yaml

If the loadgen receiver needs to be included in a very small collector, I would suggest moving the `loadgen` directory under the `./distributions` one.

Reply:
Done in b649fad

Reply:
> I'm fine with using it if it isn't used for anything critical.

Anything, we're not releasing any package/binary of it.